packages/node_modules/@webex/internal-plugin-ediscovery/src/ediscovery.js

```diff
@@ -14,15 +15,24 @@ import ReportGenerator from './report-generator';
 const EDiscovery = SparkPlugin.extend({
   namespace: 'EDiscovery',
 
+  session: {
+    contentSummaryCache: {
+      type: 'object',
+      default() {
+        return new Map();
+      }
+    }
+  },
+
```

Comment:
New contentSummaryCache property to cache space summary info which is used repeatedly while a report is generated